web/src/App.js

```diff
@@ -636,7 +636,7 @@ class App extends Component {
                 {
                   this.renderAccount()
                 }
-                <SelectLanguageBox />
+                {this.state.account && <SelectLanguageBox languages={this.state.account.organization.languages} />}
```

Comment:
The user is not logged in yet. Use `application.organizationObj`

Reply:
A &&  B is equel 

`
if ( A !== "null" ||  A !== undefined){
    B
}
`

And the `<SelectLanguageBox />` is after logged in.